doc/data/messages/i/invalid-metaclass/good.py

```diff
@@ -1 +1,2 @@
-# This is a placeholder for correct code for this message.
+class ValidApple(type):
```

Comment:
```suggestion
class Plant:
    pass

class Apple(Plant):
```

Reply:
Do you agree with this suggestion @clavedeluna ?

Reply:
Yep! My only thought when seeing this is that if one of the goals of this docs project is to outsource to contributors adding these examples, then maybe the variety that contributors bring to the table is worth keeping. I don't mind either way.